bindings/icub.i

```diff
@@ -93,6 +101,33 @@ using namespace yarp::sig;
 %include <iCub/skinDynLib/skinContactList.h>
 
 
+%typemap(in, numinputs=0) (yarp::sig::Matrix &H, yarp::sig::Vector &s, double &error) {
+    yarp::sig::Matrix t1;
+    yarp::sig::Vector t2;
+    double t3;
+    $1= &t1;
+    $2= &t2;
+    $3= &t3;
+}
+
```

Comment:
I'd add something like 
```
#ifdef SWIG_PYTHON
..... 
#endif
```
I am afraid that swig will not like this "python" directive while trying to compile bindings for other languages

Reply:
:+1:  Thanks @Nicogene . Changes are pushed